integration-test/helpers/pages.js

```diff
@@ -242,7 +263,8 @@ export function loginPage (page, server, opts = {}) {
          */
         async assertParentOpened () {
             const credsCalls = await mockedCalls(page, ['getSelectedCredentials'], true)
-            expect(credsCalls.length).toBe(5)
+            const hasSucceeded = credsCalls.some((call) => call[2]?.some(({type}) => type === 'ok'))
+            expect(hasSucceeded).toBe(true)
```

Comment:
This was also cause for flaky tests, because depending on the speed of your machine the number could fluctuate. Again, what we want is to make sure that at the call succeeded, regardless of how many polls happened.